examples/experimental/dagster-airlift/README.md

```diff
@@ -1,26 +1,178 @@
+    )
+)
+```
+
+Note: When a code server is loaded for the first time it will query the target Airflow REST API. Subsequent process loads (e.g. a run worker loading) will used a cached response. If you want Dagster to pick up new DAGs, you will need restart the code server.
+
+An MWAA auth backend is available at `dagster_airlift.mwaa.auth.MwaaSessionAuthBackend`
+
+This function creates:
+
+- An asset representing each DAG. This asset is “materialized” whenever a DAG run completes.
+- A sensor that polls the Airflow instance for operational information. This is what keeps the DAG execution history up to date. It must be on to get timely information.
+
+### Peering to multiple instances
+
+Airlift supports peering to multiple Airflow instances, as you can invoke `create_airflow_instance_defs` multiple times and combine them with `Definitions.merge`:
+
+```python
+from dagster import Definitions
+
+from dagster_airlift.core import AirflowInstance, build_defs_from_airflow_instance
+
+defs = Definitions.merge(
+    build_defs_from_airflow_instance(
+        airflow_instance=AirflowInstance(
+            auth_backend=BasicAuthBackend(
+                webserver_url="http://yourcompany.com/instance_one",
+                username="admin",
+                password="admin",
+            ),
+            name="airflow_instance_one",
+        )
+    ),
+    build_defs_from_airflow_instance(
+        airflow_instance=AirflowInstance(
+            auth_backend=BasicAuthBackend(
+                webserver_url="http://yourcompany.com/instance_two",
+                username="admin",
+                password="admin",
+            ),
+            name="airflow_instance_two",
+        )
+    ),
+)
+```
+
+## Observing Assets
+
+The next step is to observe the assets that are orchestrated from Airflow. In order to do that we must create the corresponding definitions in the Dagster deployment.
+
+We have an included example at `examples/experimental/dagster-airlift/examples/peering-with-dbt`. We suggest mimicking the structure of this project as a starting point.
+
+To add definitions that an Airlift-enabled deployment will observed, you need to use the `orchestrated_defs` argument to `build_defs_from_airflow_instance`
+
+_Note: This also accepts an object of type `Definitions`. We have recently added a function, `Definitions.merge`, which allows users to combine and compose `Definitions` objects, which this utilizes._
+
+For most projects we anticipate the following process:
+
+- Create a new factory class for every operator type or specialized use case of an operator (e.g. a BashOperator invoking a dbt project).
+- Use that factory to create definitions associating with each Airflow task.
+
+In our example we have a pre-written factory class (in `examples/experimental/dagster-airlift/dagster_airlift/dbt/multi_asset.py` and installable by `uv pip install dagster-airlift[dbt]`) that is associated with invoking a dbt project:
+
+```python
+@dataclass
```

Comment:
add import of `dagster_airlift.core.DefsFactory` (also worth ensuring that's exported top level)